packages/flutterfire_cli/lib/src/commands/config.dart

```diff
@@ -98,9 +106,26 @@ class ConfigCommand extends FlutterFireCommand {
   }
 
   String? get androidApplicationId {
-    final value = argResults!['android-app-id'] as String?;
-    // TODO validate appId is valid if provided
-    return value;
+    final value = argResults!['android-package-name'] as String?;
+
+    final deprecatedValue = argResults!['android-app-id'] as String?;
+
+    // TODO validate appId is valid if provided.
+
+    if (value != null) {
+      return value;
+    }
+    if (deprecatedValue != null) {
+      logger.stdout(
+        'Warning - android-app-id (-a) is deprecated. Consider using android-package-name (-p) instead.',
+      );
+      return deprecatedValue;
+    }
+
+    throw FirebaseCommandException(
+      'configure',
+      'Please provide value for android-package-name.',
+    );
```

Comment:
```suggestion
    if (isCI) {
      throw FirebaseCommandException(
        'configure',
        'Please provide value for android-package-name.',
      );
    }
```

since we only want to error on CI where there is no interactivity (so we can't prompt to input a value)